components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx

```diff
@@ -203,12 +217,22 @@ export const PrebuildDetailPage: FC = () => {
                             </div>
                             <div className="h-112 border-pk-border-base">
                                 <Suspense fallback={<div />}>
-                                    <WorkspaceLogs
-                                        classes="h-full w-full"
-                                        xtermClasses="absolute top-0 left-0 bottom-0 right-0 mx-6 my-0"
-                                        logsEmitter={logEmitter}
-                                        isLoading={isStreamingLogs}
-                                    />
+                                    {logNotFound ? (
+                                        <div className="px-6 py-4 h-full w-full bg-pk-surface-primary text-base flex items-center justify-center">
+                                            <Text className="w-[22rem] text-center">
+                                                Logs of failed prebuilds are inaccessible. Use{" "}
```

Comment:
Remove `failed` in case logs got deleted

Reply:
@mustard-mh "Logs of prebuilds are inaccessible" does not sound very good. Do you think it would be better to use "Logs of this prebuild are inaccessible"? I understand that we could want to show it otherwise, but the error message becomes less useful, and again it will start seeming like an error case. 
For now, we only show the message if the prebuild failed. Should we also change to be any case of not found?

Reply:
👍  Yes, like this comment https://github.com/gitpod-io/gitpod/pull/19460#discussion_r1502198031. Change message to `Logs of this prebuild ...` looks better.

Reply:
Fixed in https://github.com/gitpod-io/gitpod/pull/19460/commits/11b33f6ffda38a6b18b29e35ca248e956bcd121a